renovate.json

```
@@ -0,0 +1,12 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+      "config:base"
+    ],
+    "enabled": true,
+    "addLabels": ["dependency update"],
+    "reviewers": ["KinsonDigitalAdmin"],
+    "assignees": ["CalvinWilkinson"],
+    "commitBodyTable": true,
+    "baseBranches": ["develop"]
+}
```

Change the config to the setup below to take advantage of the org level config.

```suggestion
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "extends": ["github>KinsonDigital/.github//config/renovate-config.json"],
    "baseBranches": ["develop", "preview/v[0-9]+.[0-9]+.[0-9]+-preview.[0-9]+"]
}
```